content/docs/hooks-state.md

```diff
@@ -1,38 +1,38 @@
 ---
 id: hooks-state
-title: Using the State Hook
+title: A State Horog használata
 permalink: docs/hooks-state.html
 next: hooks-effect.html
 prev: hooks-overview.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+A *Horog* egy új kiegészítés a React 16.8-as verziójától kezdve. Ennek segítségével osztályok nélkül is használhatsz belső állapotot és egyéb React funkciókat.
```

Comment:
This part is consequently translated on several pages, as follows:
```suggestion
A *Horgok* a React egy új kiegészítése a 16.8-as verziótól kezdve. Lehetővé teszik számodra állapotok és más React funkciók használatát osztályok írása nélkül.
```

If you agree, please accept this suggestion, otherwise, I would like to ask you to fix existing translations with your version, so we keep it the same.